url_handlers.go

```diff
@@ -96,6 +96,33 @@ func WaitListHandler(w http.ResponseWriter, r *http.Request) {
   RenderTemplate(w, "assets/templates/waitlist.html.tmpl", nil)
 }
 
+func isAssignedBuzzerHandler(w http.ResponseWriter, r *http.Request) {
+    returnObj := map[string] interface{} {"status": false}
```

Comment:
Can we change this so that the `status` field indicates whether or not the request itself is successful instead of whether or not a party has been assigned a buzzer?

I think the general structure of JSON objects returned by API endpoints should be something like:

If the API call failed:
`{"status" : "error", "error_message": "Provided JSON object is invalid, Missing parameter, No party found with that id, etc"}`

If the API call succeeded:
`{"status": "success", "is_party_assigned_buzzer": false}`

I'm open to suggestions but I think this is a good way to structure API  responses.